tests/monotouch-test/AudioUnit/AudioUnitTest.cs

```diff
@@ -49,6 +49,24 @@ public void GetElementCount ()
 			var mixer = graph.GetNodeInfo (mixerNode);
 			Assert.AreEqual (1, mixer.GetElementCount (AudioUnitScopeType.Global));
 		}
+
+		[Test]
+		public void CopyIconTest ()
+		{ 
+			AudioComponentDescription cd = new AudioComponentDescription () {
+				ComponentType = AudioComponentType.Output,
+#if MONOMAC
+				ComponentSubType = (int)AudioUnitSubType.VoiceProcessingIO,
+#else
+				ComponentSubType = 0x72696f63, // Remote_IO
+#endif
+				ComponentManufacturer = AudioComponentManufacturerType.Apple
+			};
+			AudioComponent component = AudioComponent.FindComponent (ref cd);
+			Assert.DoesNotThrow ( () => {
+				var icon = component.CopyIcon (); // ensuring that the manual binding does not throw, we do not care about the result
+			});
```

Comment:
You could do something like this:

```csharp
Assert.IsNotNull (component.CopyIcon (), "CopyIcon");
```

Reply:
It can be null, there is no guarantee, I much prefer to make sure it does not crash.